Yazi enables vendored Lua in every workspace crate. Disable those defaults so
Void's packaged Lua 5.5 is used instead.

--- a/yazi-actor/Cargo.toml
+++ b/yazi-actor/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-binding/Cargo.toml
+++ b/yazi-binding/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-core/Cargo.toml
+++ b/yazi-core/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-dds/Cargo.toml
+++ b/yazi-dds/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-fm/Cargo.toml
+++ b/yazi-fm/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-parser/Cargo.toml
+++ b/yazi-parser/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-plugin/Cargo.toml
+++ b/yazi-plugin/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-runner/Cargo.toml
+++ b/yazi-runner/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-scheduler/Cargo.toml
+++ b/yazi-scheduler/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-shim/Cargo.toml
+++ b/yazi-shim/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
--- a/yazi-widgets/Cargo.toml
+++ b/yazi-widgets/Cargo.toml
@@ -13,7 +13,6 @@
 workspace = true
 
 [features]
-default      = [ "vendored-lua" ]
 vendored-lua = [ "mlua/vendored" ]
 
 [dependencies]
